### ISSUE TITLE
Giveaway keywords

### ISSUE THREAD

**Issue by MLWALK3R** [Vedzaa/NitroSniperGo#14]
Have an option for keywords (nitro, beta, key, etc) ranter than a catch all; it will enter you into some risky stuff ;)

**Comment by Vedzaa** [comment]
That's a good idea, you can edit it in the source code but if I have time I will do something like that

**Comment by Vedzaa** [comment]
I updated to only join Nitro giveaway as it's the point of the program for the time being.

**Comment by MLWALK3R** [comment]
I tried that code last night and it would still enter any giveaway regardless.

**Comment by MLWALK3R** [comment]
```
17:58:02 [+] Bot is ready
<:yay:585696613507399692>   **GIVEAWAY**   <:yay:585696613507399692>
17:59:14 [-] Enter Giveaway  [haHAA > general]
17:59:25 [+] Won Giveaway: tester [haHAA > general]
```

**Comment by Vedzaa** [comment]
You're right, let me fix it

**Comment by Vedzaa** [comment]
Should be fixed ee92ebc8456512b74293a80b02250afdc2f62f67

**Comment by Vedzaa** [comment]
I close the issue and keep your idea in mind, if someone doesn't want the last version use [this one](https://github.com/Vedzaa/NitroSniperGo/tree/6df969383918accd19389978dacbbce61396a82d)
